project_management/workshop_advert.md

```diff
@@ -0,0 +1,56 @@
+## Boost your research reproducibility with Binder
+
+If you are looking for an easy way to share your code with others, without worrying about what software they have, then myBinder.org might be the solution for you. Binder allows you to share your software with others in the form of a single clickable link, making it easier for users and removing the need for you to worry about supporting lots of different platforms. 
+
+During this free workshop we will discuss reproducible computing environments, show examples of others’ projects in myBinder.org and help you learn how to prepare a binder-ready project. At the end of the workshop you will be able to take some of your own content (in a R or Jupyter notebook) and prepare it so that it can be used by others on myBinder.org.
```

Comment:
It doesn't necessarily _have_ to be an R or Jupyter notebook, it can work with scripts too. I would maybe eliminate that so people don't feel like they have to change the format of their code and it's an easy switch to a JupyterLab or RStudio environment if needs be.

Reply:
Ok, will do, thanks for clarifying for the non-coder 😃

Reply:
My suggestion:

> (for example as an R markdown document, Jupyter notebook, or scripts that can be run in the terminal)